src/EditorFeatures/Core/InlineHints/InlineHintsDataTaggerProvider.cs

```diff
@@ -101,6 +101,12 @@ protected override IEnumerable<SnapshotSpan> GetSpansToTag(ITextView? textView,
             if (document == null)
                 return;
 
+            // The LSP client will handle producing tags when running under the LSP editor.
+            // Our tagger implementation should return nothing to prevent conflicts.
+            var workspaceContextService = document.Project.Solution.Services.GetRequiredService<IWorkspaceContextService>();
+            if (workspaceContextService?.IsInLspEditorContext() is true)
```

Comment:
```suggestion
            if (workspaceContextService.IsInLspEditorContext() is true)
```

its GetRequiredService above so it shouldn't be null